### PR TITLE
Migrate geo_json_events platform to integration with config flow

### PR DESCRIPTION
**Pull request by exxamalte**
## Proposed change

Adapted documentation to reflect proposed changes of the integration to config flow. Removed YAML based configuration examples. Made small enhancements to the existing documentation to align with other similar integrations.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/87062
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by exxamalte** [comment]
Just rebased to latest `next` branch; no changes to content.

File: source/_integrations/geo_json_events.markdown [hunk 1]
```diff
@@ -5,74 +5,50 @@ ha_category:
...
   - '@exxamalte'
+ha_domain: geo_json_events
 ha_platforms:
   - geo_location
+  - sensor
 ha_integration_type: integration
 ---
 
-The `geo_json_events` platform lets you integrate GeoJSON feeds. It retrieves events from a feed and shows information of those events filtered by distance to Home Assistant's location.
+The `geo_json_events` integration lets you ingest events from GeoJSON feeds. It retrieves events from a feed and shows information of those events filtered by distance to Home Assistant's location.
```
Comment (frenck) [inline source/_integrations/geo_json_events.markdown]:
```suggestion
The GeoJSON integration lets you ingest events from GeoJSON feeds. It retrieves events from a feed and shows information of those events filtered by distance to Home Assistant's location.
```

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by frenck — DISMISSED** [review]

**Comment by exxamalte** [comment]
I reduced the scope of the parent PR in the codebase to only migrate to config flow, so the latest change in the documentation reflects that.

**Review by frenck — APPROVED** [review]

**Pull request merged**